tensorflow-framework/src/main/java/org/tensorflow/framework/activations/Exponential.java

```diff
@@ -31,16 +32,59 @@
  *   // result is [0.04978707f,  0.36787945f,  1.f,  2.7182817f, 20.085537f]
  * </pre>
  */
-public class Exponential<T extends TFloating> extends AbstractActivation<T> {
+public class Exponential extends AbstractActivation {
+  /** The activation name as known by TensorFlow */
+  public static final String NAME = "exponential";
 
   /** Creates an Exponential activation. */
   public Exponential() {
     super();
   }
 
+  /**
+   * Creates a new Exponential from a configuration Map
+   *
+   * @param config the configuration map, this class does not use any of the entries in the
+   *     configuration map
+   */
+  public Exponential(Map<String, Object> config) {
+    this();
+  }
+
+  /**
+   * Computes the Exponential activation function.
+   *
+   * <p>Example Usage:
+   *
+   * <pre>
+   *      Operand&lt;TFloat32&gt; input = &#46;&#46;&#46;;
+   *      Operand&lt;TFloat32&gt; result = Exponential.exponential(tf, input);
+   * </pre>
+   *
+   * @param tf the TensorFlow Ops
+   * @param input the input
+   * @param <T> the data type for the input
+   * @return the exponential activation: {@code exp(x)}.
+   */
+  public static <T extends TNumber> Operand<T> exponential(Ops tf, Operand<T> input) {
```

Comment:
Why is the bound `TNumber` rather than `TFloating`?

Reply:
TF Python, tf.math.exp requires floating, so I will fix.

Reply:
Using `TNumber` works, but `TFloating` creates an error.
`'exponential(org.tensorflow.op.Ops, org.tensorflow.Operand<T>)' in 'org.tensorflow.framework.activations.Exponential' cannot be applied to '(org.tensorflow.op.Ops, org.tensorflow.Operand<T>)'`

Reply:
Actually the problem here is that `public <T extends TNumber> Operand<T> call(Ops tf, Operand<T> input) `
is part of the functional interface for all activations. I have this method calling the underlying static method, `exponential()`, using the state from the Exponential class. So if the static method,`exponential()`, is `TFloating`, then it is out of sync with `TNumber` in the `call() `method. 

For now, I have the  `call()` method invoking `tf.math.exp()` directly,  the way it used to be,  and I changed the signature for `exponential()` to use `TFloating`.